packages/expo-modules-core/ios/AppDelegates/ExpoAppInstance.swift

```diff
@@ -0,0 +1,94 @@
+import React_RCTAppDelegate
+
+private var reactDelegateHandlers = [ExpoReactDelegateHandler]()
+
+@objc(EXAppInstance)
+open class ExpoAppInstance: RCTAppDelegate {
+  @objc
+  public let reactDelegate = ExpoReactDelegate(handlers: reactDelegateHandlers)
+
+  @objc
+  open override func createRootViewController() -> UIViewController {
+    return reactDelegate.createRootViewController()
+  }
+
+  open override func bundleURL() -> URL? {
+#if DEBUG
+    return RCTBundleURLProvider.sharedSettings().jsBundleURL(forBundleRoot: ".expo/.virtual-metro-entry")
```

Comment:
@vonovak Hello, I use Bare ReactNative project.

I updated react-native from 0.76.6 to 0.77.0

It works with expo 52.0.25

But if I updating expo from 52.0.25 to 52.0.26 I see error on iOS app start:

Unable to resolve module ./.expo/.virtual-metro-entry from

Reply:
Fixed in expo 52.0.28. Should be in https://github.com/expo/expo/pull/34464